packages/curve-ui-kit/src/widgets/Footer/Sections.tsx

```diff
@@ -75,7 +75,7 @@ export const getSections = (): SectionProps[] => [
       },
       {
         label: t`Integrations`,
-        href: '#/integrations',
+        href: window.location.origin + '#/integrations',
```

Comment:
this might crash on server-side renders (no window)
this will also ignore the react router and force a full page refresh